### PR TITLE
[BE-1] Tip Of The Day

### PR DESCRIPTION
**Pull request by fangherk**
Create an initial endpoint to receive daily tasks @ `http://localhost:3000/tips_daily/X`.

This returns a json in the format of 

```
{"id":150,
"tip":"Introduce green space into your everyday work environment.",
"info":"They filter pollutants and dust from the air, and can provide moisture to the air that lowers overall temperature for a comfortable work environment.",
"link":"",
"category":"Food (organic, non-GMO)",
"created_at":"2018-07-04T05:38:52.932Z",
"updated_at":"2018-07-04T05:38:52.932Z"}
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This repo contains the backend for various Turning Green stuff.
 
-Things you may want to cover:
+## Initial Setup
+For first-time users, do the following
+```
+1. Get Docker
+2. `docker-compose run web rails db:create`
```
Comment (dolumese) [inline README.md]: @fangherk Reading online it seems better to run `docker-compose build` before 2.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+3. `docker-compose run web rails db:migrate`
+4. `docker-compose up`
```
Comment (dolumese) [inline README.md]: Need to add `docker-compose run web rake task:importjson` before starting up the server

File: docker-compose.yml [hunk 3]
```diff
@@ -5,6 +5,8 @@ services:
     volumes:
       - ./tmp/db:/var/lib/postgresql/data
```
Comment (dolumese) [inline docker-compose.yml]: This is the guy that cause me much time and debugging.
As is, this causes as error on my machine when setting up the postgresql database: `could not translate host name "db" to address: Name or service not known`.

https://www.linux.com/learn/docker-volumes-and-networks-compose: suggests that named volumes are used instead. Bind volumes can be used if you want to bind a specific file into the docker container. (https://stackoverflow.com/questions/41637505/how-to-persist-data-in-a-dockerized-postgres-database-using-volumes)

So, we can change this to `db-data:/var/lib/postgresql/data`, for example. Or any other name. And then at the bottom of the file add:
```yml
volumes:
  db-data:
```

**Review by dolumese — CHANGES_REQUESTED** [review]
It works on my machine with the suggested changes. Please confirm with yours

File: docker-compose.yml (hunk 3 above)
Comment (fangherk) [reply docker-compose.yml]: This works.

**Review by dolumese — APPROVED** [review]

**Pull request merged**